### PR TITLE
Latest build from Swagger

### PR DESCRIPTION
**Pull request by tspence**
Includes the recent improvements to the Java generator

### CODE REVIEW

File: src/main/java/io/lockstep/api/LockstepApi.java [hunk 1]
```diff
@@ -42,10 +42,7 @@
 import io.lockstep.api.clients.UserRolesClient;
 
 /**
- * Represents a failed request.
- *
- * @author jkirk
```
Comment (jkirk-lockstep) [inline src/main/java/io/lockstep/api/LockstepApi.java:47]: As we discussed this makes sense.  I agree the author should not change constantly.

Comment (tspence) [reply src/main/java/io/lockstep/api/LockstepApi.java:47]: Thanks, I agree

File: pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
     <groupId>io.lockstep</groupId>
     <artifactId>lockstepsdk</artifactId>
-    <version>2021.39.690</version>
+    <version>2022.2.93.0</version>
```
Comment (jkirk-lockstep) [inline pom.xml:8]: I see this now matches the API version pattern.  Agree this is a good plan.

Comment (tspence) [reply pom.xml:8]: 👍

**Comment by jkirk-lockstep** [comment]
Thank you for all the Javadoc work!  This should improve usability greatly.

**Pull request closed without merging**